Corona-Warn-App/build.gradle

```diff
@@ -440,6 +441,8 @@ dependencies {
     implementation 'com.google.guava:guava:30.0-android'
     implementation 'joda-time:joda-time:2.10.6'
 
+    implementation 'com.networknt:json-schema-validator:1.0.56'
```

Comment:
I picked this library as it seems well maintained and  internally uses jackson which we will get as dependency via https://github.com/eu-digital-green-certificates/dgc-certlogic-android anyways.